scripts/tools_assure_gogenerate.sh

```diff
@@ -4,7 +4,8 @@ go get -u golang.org/x/tools/cmd/stringer github.com/tinylib/msgp
 cd $GOPATH/src/github.com/raintank/metrictank/
 # stringer needs all packages to be installed to work
 # see https://github.com/golang/go/issues/10249
-go install ./...
+#go install ./...
+go install github.com/raintank/metrictank/vendor/github.com/hashicorp/memberlist github.com/raintank/metrictank/vendor/github.com/dgryski/go-tsz 
```

Comment:
@woodsaj i haven't been able to get stringer to properly run. you can see the errors in circleCI. so while this PR seems to work fine for msgp, it's not reliable yet for stringer